docs/sql-ref-syntax-ddl-create-table-hiveformat.md

```diff
@@ -51,15 +59,55 @@ as any order. For example, you can write COMMENT table_comment after TBLPROPERTI
 
 * **EXTERNAL**
 
-    Table is defined using the path provided as LOCATION, does not use default location for this table.
+    Table is defined using the path provided as `LOCATION`, does not use default location for this table.
 
 * **PARTITIONED BY**
 
     Partitions are created on the table, based on the columns specified.
+    
+* **row_format**    
+
+    Use the `SERDE` clause to specify a custom SerDe for one table. Otherwise, use the `DELIMITED` clause to use the native SerDe and specify the delimiter, escape character, null character, and so on.
+    
+* **SERDE**
+
+    `SERDE` clause can be used to specify a custom `SerDe` for one table.
+    
+* **serde_class**
+
+    Specify a fully-qualified class name of a custom SerDe.
+
+* **SERDEPROPERTIES**
+
+    A list of key-value pairs that is used to tag the serde definition.
+    
+* **DELIMITED**
+
+    `DELIMITED` clause can be used to specify the native `SerDe` and state the delimiter, escape character, null character, and so on.
+    
+* **FIELDS TERMINATED BY**
 
-* **ROW FORMAT**
+    It is used to define a column separator.
```

Comment:
remove `it is`?
It is used to define a column separator -> Used to define a column separator?
Same for the next a few sentences.